content/tutorial/02-advanced-svelte/04-actions/01-actions/app-b/src/lib/App.svelte

```diff
@@ -2,7 +2,7 @@
 	import Canvas from './Canvas.svelte';
 	import { trapFocus } from './actions.js';
 
-	const colors = ["rouge", "orange", "jaune", "vert", "bleu", "indigo", "violet", "blanc", "noir"].;
+	const colors = ['red', 'orange', 'yellow', 'green', 'blue', 'indigo', 'violet', 'white', 'black'];
```

Comment:
C'est volontaire ? C'est le point qui posait problème peut-être ?
```suggestion
	const colors = ["rouge", "orange", "jaune", "vert", "bleu", "indigo", "violet", "blanc", "noir"];
```

Reply:
Oui, le point posait problème, et aussi, ici ce sont des couleurs CSS, du coup il faut les garder en anglais.